wgpu/src/shader/triangle/solid.wgsl

```diff
@@ -25,6 +25,6 @@ fn vs_main(input: VertexInput) -> VertexOutput {
 }
 
 @fragment
-fn fs_main(input: VertexOutput) -> @location(0) vec4<f32> {
+fn fs_main(input: SolidVertexOutput) -> @location(0) vec4<f32> {
     return input.color;
 }
```

Comment:
Should we unify `triangle/gradient.wgsl` and `triangle/solid.wgsl` into `triangle.wgsl` like `quad.wgsl`?